bentoml/deployment/store.py

```diff
@@ -16,45 +16,92 @@
 from __future__ import division
 from __future__ import print_function
 
+import logging
+
 from sqlalchemy import Column, String, JSON
-from google.protobuf.json_format import MessageToDict
+from sqlalchemy.orm.exc import NoResultFound, MultipleResultsFound
+from google.protobuf.json_format import MessageToDict, ParseDict
 
+from bentoml.config import config
 from bentoml.db import Base, create_session
+from bentoml.proto import deployment_pb2
+
+
+logger = logging.getLogger(__name__)
 
 
 class Deployment(Base):
     __tablename__ = 'deployments'
 
     id = Column(String, primary_key=True)
     name = Column(String, unique=True)
-    namespace = Column(String)
+    namespace = Column(String, nullable=False)
 
     spec = Column(JSON)
-    labels = Column(JSON)
-    annotation = Column(JSON)
+    labels = Column(JSON, nullable=False, default={})
```

Comment:
I think labels and annotation fields are optional, correct?  If they are optional, then nullable won't need to set to be False

Reply:
Will need to test out how it works when doing filtering by label on null jsonb filed in postgres

Reply:
they are indeed optional, note the default value '{}'